src/components/molecules/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,59 @@
+import VenueChat from "components/molecules/VenueChat";
+import ChatsList from "components/molecules/ChatsList";
+import LiveSchedule from "components/molecules/LiveSchedule";
+import { useSelector } from "hooks/useSelector";
+import { useVenueId } from "hooks/useVenueId";
+import React, { useState } from "react";
+import { useFirestoreConnect } from "react-redux-firebase";
+import "./Sidebar.scss";
+
+const Sidebar = () => {
+  const venueId = useVenueId();
+  useFirestoreConnect({
+    collection: "users",
+    where: ["enteredVenueIds", "array-contains", venueId],
+    storeAs: "chatUsers",
+  });
```

Comment:
I'm not 100% about this.. but the other day @thecadams and I were thinking that memo'ing the config for `useFirestoreConnect` may contribute to avoiding re-renders/etc.

I've since updated my theory a little in that it may be more related to our usages of selectors, so not 100% if it is required.. but if you can think of an easy way to isolate and test if we need to memo the config for this (or in this case, just extract it to a static `const` outside of the component), that would be really useful information to know/evidence for.

Reply:
I ended up looking into this as part of work on https://github.com/sparkletown/sparkle/pull/721 / https://github.com/sparkletown/sparkle/pull/726 and found that the config doesn't need to be memo'd by us, as `useFirestoreConnect` does it for us ([internal ref](https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1605594256020600))

Essentially, it does a deep compare using lodash's `isEqual`.